ldm/generate.py

```diff
@@ -649,6 +649,7 @@ def _make_txt2img(self):
         if not self.generators.get('txt2img'):
             from ldm.dream.generator.txt2img import Txt2Img
             self.generators['txt2img'] = Txt2Img(self.model, self.precision)
+            self.generators['txt2img'].free_gpu_mem = self.free_gpu_mem
```

Comment:
This is a bit awkward, but there's no other way to do it in the current model without adding more arguments to its __init__ method and then making the corresponding changes in generate.py.